src/runtime-rs/crates/hypervisor/src/device/driver/virtio_net.rs

```diff
@@ -20,20 +28,106 @@ impl fmt::Debug for Address {
     }
 }
 
-#[derive(Debug, Clone)]
+#[derive(Clone, Debug, Default)]
 pub struct NetworkConfig {
+    pub index: u64,
```

Comment:
Add a one line comment doc for this field please.

Reply:
fixed